frontend/src/pages/projects/screens/projects/useProjectTableRowItems.ts

```diff
@@ -18,40 +21,72 @@ const useProjectTableRowItems = (
   isRefreshing: boolean,
   setEditData: (data: ProjectKind) => void,
   setDeleteData: (data: ProjectKind) => void,
-): KebabItem[] => {
-  const [allowCreate] = useAccessReview({
-    ...accessReviewResource,
-    namespace: project.metadata.name,
-  });
+): [KebabItem[], () => void] => {
   const navigate = useNavigate();
+  const [shouldRunCheck, setShouldRunCheck] = React.useState(false);
+
+  const [allowUpdate, allowUpdateLoaded] = useAccessReview(
+    {
+      ...accessReviewResource,
+      namespace: project.metadata.name,
+      verb: 'update',
+    },
+    shouldRunCheck,
+  );
+  const [allowCreate, allowCreateLoaded] = useAccessReview(
+    {
+      ...accessReviewResource,
+      namespace: project.metadata.name,
+    },
+    shouldRunCheck,
+  );
+  const [allowDelete, allowDeleteLoaded] = useAccessReview(
+    {
+      ...accessReviewResource,
+      namespace: project.metadata.name,
+      verb: 'delete',
+    },
+    shouldRunCheck,
+  );
+
+  const runAccesCheck = React.useCallback(() => {
+    setShouldRunCheck(true);
+  }, []);
+
+  const noPermissionToolTip = (allow: boolean, loaded: boolean) =>
```

Comment:
Technically a nit because our linter should be doing this if we wanted it...

But typically you always want to type functions for both input and return -- in this case, it's a little less important, but I think it's a good practice to get into.

If you create this function in the root, I believe our linter yells at you to add a return type.

```suggestion
  const noPermissionToolTip = (allow: boolean, loaded: boolean): Partial<KebabItem> =>
```

Reply:
Sure thing, I just added the `Partial<KebabItem> | undefined` return type onto it. Oddly the linter doesn't complain if the function is at the root either.

Reply:
Interesting... I swore we had that linter. cc @christianvogt ?

Reply:
Looks like the eslint rule is disabled for that 
in `frontend/.eslintrc` on line 156 
```
    "@typescript-eslint/explicit-function-return-type": "off",
```

Reply:
We do have this rule enabled: `@typescript-eslint/explicit-module-boundary-types`
This forces a type for all exported properties. Anything else internal to the module can use inference.